<launch>
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find lane_line)/config/custom_rosconsole.conf"/>

    <env name="GLOG_minloglevel" value="1"/>

    <node pkg="lane_line" type="lane_line" name="lane_line" output="screen"> <!-- screen/log -->
        <rosparam command="load" file="$(find lane_line)/config/gantry.yaml" />
    </node>
</launch>
